io_test.go

```diff
@@ -221,8 +340,13 @@ func TestOutput(t *testing.T) {
 			cfg.RegularGrid(ctmdata, pop, popIndices, mr, emis),
 		},
 		CleanupFuncs: []DomainManipulator{
-			Output(TestOutputFilename, false, "TotalPop deaths", "TotalPop",
-				"TotalPM25", "PM2.5 emissions", "Baseline TotalPM25", "WindSpeed"),
+			Output(TestOutputFilename, false, map[string]string{
+				"TotalPopDeaths":    "coxHazard(loglogRR(TotalPM25), TotalPop, MortalityRate)",
+				"TotalPop":          "TotalPop",
+				"TotalPM25":         "TotalPM25",
+				"PM25Emissions":     "PM25Emissions",
+				"BaselineTotalPM25": "BaselineTotalPM25",
```

Comment:
That's good, but then how is it that there are more than 10 characters in "BaselineTotalPM25" but it still works?